src/Kobo/SystemDialog.cpp

```diff
@@ -87,9 +91,25 @@ SystemWidget::Prepare(ContainerWindow &parent, const PixelRect &rc) noexcept
 inline void
 SystemWidget::SwitchOTGMode()
 {
-#ifdef KOBO
```

Comment:
Will this still compile without the `#ifdef`?

Reply:
It compiles fine on my development machine.  I compile with TARGET=KOBO,

Reply:
I will put back the ifdefs as they may be needed for UNIX... TARGETs.

Reply:
ifdef is now back